Tribler/Test/test_torrent_checking.py

```diff
@@ -36,4 +35,22 @@ def test_torrent_checking(self):
 
         num_seeders = torrent['num_seeders']
         num_leechers = torrent['num_leechers']
-        assert num_leechers >= 0 or num_seeders >= 0, "No peers found: leechers: %d seeders: %d" % (num_leechers, num_seeders)
+        assert num_leechers >= 0 or num_seeders >= 0, "No peers found: leechers: %d seeders: %d" % (
+        num_leechers, num_seeders)
+
+    def test_udp_torrent_checking(self):
+        tdef = TorrentDef.load(TORRENT_FILE)
+        tdef.set_tracker("udp://localhost")
+        tdef.metainfo_valid = True
+
+        self.tdb.addExternalTorrent(tdef)
+        self.session.check_torrent_health(tdef.get_infohash())
+        sleep(31)
```

Comment:
Oh man, it's a Monday alright! That's a copy of the test over it... Those two tests are responsible for a full minute of test execution time. That's ridiculous.